app/controllers/report_controller.rb

```diff
@@ -178,6 +178,10 @@ def x_show
   def tree_select
     @edit = nil
     @sb[:select_node] = false
+    if @sb.try(:[], :flash_msg)
```

Comment:
`@sb` is being set in `ApplicationController` before every action either to an empty hash or to whatever
the sandbox for the given controller contained before. I haven't seen this `@sb.try(:[], ...)` construct anywhere
in our code, I consider it a bad coding practice and I won't merge this PR looking like this.